test/private/files.spec.ts

```diff
@@ -467,27 +467,38 @@ describe('files', () => {
 
   describe('openDownloadFolder', () => {
     it('should not allow calls before initialization', () => {
-      expect(() => files.openDownloadFolder()).toThrowError(
```

Comment:
should have unit tests that pass in a value for filepath, all of these tests seem to pass in null for filepath

Reply:
Added the unit test as suggested.